recipes/demjson3/meta.yaml

```
@@ -0,0 +1,47 @@
+{% set name = "demjson3" %}
+{% set version = "3.0.5" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: ab9aabdd85695f3684fc296f39766a2730f6c8de81d23f7048073dfe2f616d80
+  patches:
+    - 0001-Update-setup.py.patch
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - jsonlint=demjson3.jsonlint:main
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+
+test:
+  imports:
+    - demjson3
+  commands:
+    - pip check
```

```suggestion
    - pip check
    - jsonlint --help
```

I haven't verified if it has a help command but the CLI needs to be tested as well.